databases/week3/hyf_homework_databases_lesson3.sql

```diff
@@ -0,0 +1,166 @@
+-- Data model
+CREATE DATABASE hyf_meal_sharing;
+USE hyf_meal_sharing;
+
+CREATE TABLE `meal` (
+`id` int unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,
+`title` varchar(255) NOT NULL,
+`description` text NOT NULL,
+`location` varchar(255) NOT NULL,
+`when` datetime NOT NULL,
+`max_reservations` int unsigned NOT NULL,
+`price` decimal NOT NULL,
+`created_date` date NOT NULL
+);
```

Comment:
You forgot to specify the character set and the engine, see here for an example: https://github.com/HackYourFuture-CPH/databases/blob/master/lesson2/lesson2-data.sql

Same for the other two tables.